src/NHibernate.Test/NHSpecificTest/GH0750/Fixture.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using NHibernate.Criterion;
+using NHibernate.Transform;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH0750
+{
+	public class ProductSummary
+	{
+		public int ProductId { get; set; }
+
+		public string Name { get; set; }
+
+		public int TotalQuantity { get; set; }
+
+		public decimal TotalPrice { get; set; }
+	}
+
+	[TestFixture]
+	public class Fixture : BugTestCase
+	{
+		protected override void OnSetUp()
+		{
+			using (ISession session = OpenSession())
+			using (ITransaction tx = session.BeginTransaction())
+			{
+				var greenTea = new Product()
+				{
+					ProductId = 1,
+					Name = "Green Tea",
+					UnitPrice = 5
+				};
+
+				session.Save(greenTea);
+
+				var blackTea = new Product()
+				{
+					ProductId = 2,
+					Name = "Black Tea",
+					UnitPrice = 10
+				};
+
+				session.Save(blackTea);
+
+				var greenTeaOrder = new Order()
+				{
+					OrderId = 1,
+					OrderDate = System.DateTime.Now
+				};
+
+				session.Save(greenTeaOrder);
+
+				greenTeaOrder.OrderLines.Add(new OrderLine() { Order = greenTeaOrder, Product = greenTea, Quantity = 2, UnitPrice = greenTea.UnitPrice ?? 0 });
+
+				session.Save(greenTeaOrder);
+
+				var blackTeaOrder = new Order()
+				{
+					OrderId = 2,
+					OrderDate = System.DateTime.Now
+				};
+
+				session.Save(blackTeaOrder);
+
+				blackTeaOrder.OrderLines.Add(new OrderLine() { Order = blackTeaOrder, Product = blackTea, Quantity = 5, UnitPrice = blackTea.UnitPrice ?? 0 });
+
+				session.Save(blackTeaOrder);
+				tx.Commit();
+			}
+		}
+
+		protected override void OnTearDown()
```

Comment:
This was missing. Deleting all data is required by our tests, otherwise it is considered as a teardown failure.